management/instance_types/README.md

```diff
@@ -0,0 +1,68 @@
+# Scalr Policy Library
+
+## Policy :: instance_types
+
+Enforces a whitelist of allowed instance types/sizes for AWS, Azure and Google.
+
+### Files
+
+| Files | Description |
+|---|---|
+| instances_types.rego | The OPA policy |
+| instance_types.tf | Terraform config used to generate mock data for policy evaluation |
+| instance_types.input.json | Mock data (tfplan, tfrun) generated by Scalr |
+| eval.output | Expected output from running `POL=instance_types opa eval -f pretty --data ${POL}.rego -i ${POL}.input.json data.terraform.deny` |
+
+### How to use this policy
+
+#### Cloning and testing
+
+---
+
+NOTE: See [Terraform CLI with Scalr](https://docs.scalr.com/en/latest/cli.html) for details on creating an API token to enable the CLI to work with Scalr
+
+---
+
+1. Clone this repository
+2. Edit instances_types.rego as required, e.g. to adjust whitelists
+3. Replace instances_types.tf with your own Terraform configuration that you need to test the policy against. Ensure the config contains a [partial backend configuration](https://www.terraform.io/docs/language/settings/backends/configuration.html#partial-configuration) similar to this with a workspace name of your choice.
+
+```javascript
+terraform {
+
+  backend "remote" {
+    workspaces {
+      name = "{ws-name}"
+    }
+  }
+}
+```
+
+4. Set your remote backed hostname and organization (Scalr environment) parameters using `TF_CLI_ARGS_init` as follows.
+
+`export TF_CLI_ARGS_init='-backend-config="hostname=s{your-scalr-hostname}" -backed-config="organization={your-scalr-evironment_id}"`.
+
+  When using Scalr SaaS the hostname will be in the form `my-account.scalr.io`.
+  The environment_id will be in the form `env-{random_string}` and can be obtained from the environment dashboard.
+
+5. Run `terraform init` to initialize the workspace.
+6. Run `./generate_plan.sh` to generate a plan and downlaod the mock data.
```

Comment:
> @pgale61 I have google credentials configured in my environment, but still getting
> 
> ```
> Waiting for the plan to start...
> 
> Terraform v0.14.2
> Initializing Terraform configuration...
> Configuring remote state backend...
> Initializing plugins and modules...
> 
> ------------------------------------------------------------------------
> 
> 
> Error: No value for required variable
> 
>   on instance_types.tf line 380:
>  380: variable "project" {
> 
> The root module input variable "project" is not set, and has no default value.
> Use a -var or -var-file command line argument to provide a value for this
> variable.
> 
> 
> Error: Plan operation failed
> ```

Per step 3. of the readme the user would replace the supplied config with their own, so it's fair to assume they will set the provider configs as they require them. I'm not expecting users to run the supplied config, because there is a high chance it wont work for them anyway. I'll add a comment to the README.